### ISSUE TITLE
Would you share your training parameters?

### ISSUE THREAD

**Issue by hnn123** [jungomi/math-formula-recognition#2]
I had trained the model with the following parameters
![image](https://user-images.githubusercontent.com/11392660/53213640-ab5b9680-3684-11e9-8aba-de97a2f4dc41.png)
and used the dataset you shared. But I couldn't get the same result with the checkpoints you shared "xavier-gradnorm5.0-nodropout-teacher-nocrop-withpad-1-0050.pth". So would you share your training process?

**Comment by jungomi** [comment]
That was the very first attempt to make it learn properly and at that time I didn't use a train/validation split, but used all samples for training. Since it is such a small dataset, it does make quite a big difference. All the checkpoints that are shown in `notes/` used all training samples.

Regarding the parameters, they should be exactly as you have them, except for the learning rate, which started at 1e-2. The learning rates that have been used, are recorded in the checkpoints. You can check them as such:

```python
checkpoint_nodropout["lr"]
```
![checkpoint-lr](https://user-images.githubusercontent.com/3986846/53246810-f4760f80-36b1-11e9-9336-3601fefe5d7e.png)

I believe I had also not set the random seed at that time, hence you shouldn't expect the exact same results.

As for the training/validation split, I have done some other experiments, especially with variations of the architectures (e.g. LSTM instead of GRU) and in general I had used all default parameters except for the learning rate, which was mostly at 1e-2 at the beginning for about 100 - 150 epochs and then the lower rates for 50 each and I've stopped at a little over 250 epochs.

**Comment by hnn123** [comment]
Thank you very much for your reply! I will try using the learning rate you shared.
